core/src/main/scala/kafka/server/LegacyBroker.scala

```diff
@@ -129,6 +130,8 @@ class LegacyBroker(val config: KafkaConfig,
 
   var brokerMetadataListener: BrokerMetadataListener = null
 
+  var metaLogManager: MetaLogManager = null
```

Comment:
I think this stuff goes in `Kip500Broker` rather than in the Legacy/ZK broker.

Reply:
I assumed this would fit in here considering there was an instance of the `BrokerMetadataListener`. But considering this depends on the RAFT log, makes sense to make it exclusive to the `Kip500Broker`